packages/react-native/src/primitives/RadioGroup/RadioGroup.tsx

```diff
@@ -48,9 +52,10 @@ export default function RadioGroup<T>({
 
   const containerStyle: ViewStyle = useMemo(
     () => ({
+      ...themedStyle.container,
       flexDirection: getFlexDirectionFromLabelPosition(labelPosition),
     }),
-    [labelPosition]
+    [labelPosition, themedStyle]
```

Comment:
`themedStyle` is going to cause this to recalculate on every render since it does not maintain a stable reference